src/nrniv/bgpdmasetup.cpp

```diff
@@ -432,18 +433,20 @@ static int setup_target_lists(int** r_return) {
 	// target lists.
 
 	// Allocate the target lists (and set size to 0 (we will recount when filling).
-	NrnHashIterate(Int2TarList, gid2tarlist, TarList*, tl) {
+	for (auto& iter: gid2tarlist) {
+		TarList* tl = iter.second;
```

Comment:
If the value type in `gid2tarlist` becomes `unique_ptr` then this should become
```c++
auto& tl = iter.second;
```
(or `TarList* tl = iter.second.get();`)